relay-server/src/actors/outcome.rs

```diff
@@ -81,6 +81,8 @@ pub struct TrackOutcome {
     pub event_id: Option<EventId>,
     /// The client ip address.
     pub remote_addr: Option<IpAddr>,
+    /// The data categories of events in the envelope.
+    pub data_categories: Vec<DataCategory>,
```

Comment:
Every outcome should only ever have a single data category. If there were multiple categories, we should also emit multiple outcomes.

In the future, we will have to emit separate outcomes for each of the items in an envelope (that require outcomes). For example, you could be rate limiting an attachment, filtering the event, and accepting a session from a single envelope, and all with different reason codes.

For now, only the event item is relevant. Since there can only ever be a single event item (error, default, security, or transaction) in an envelope, it would be sufficient to use that and emit a single outcome.